src/pages/iou/IOUModal.js

```diff
@@ -248,6 +263,27 @@ class IOUModal extends Component {
         });
     }
 
+    /**
+     * Create the payIOU report with 'fake' reportID 0
+     *
+     * @param {Array} [splits]
+     */
+    createAndPayIOU() {
+        payIOUReport({
+            chatReportID: this.props.route.params.reportID,
+            reportID: 0,
+            paymentMethodType: this.state.paymentType,
+
+            // should send in cents to API
+            amount: Math.round(this.state.amount * 100),
+            currency: this.props.iou.selectedCurrencyCode,
+
+            // submitterPayPalMeAddress: this.props.iouReport.submitterPayPalMeAddress,
+            // submitterPhoneNumber: this.submitterPhoneNumber,
```

Comment:
@nickmurray47 Should these 2 optional parameters be included here? The `iouReport` does not exist at this point yet as I see that in [here](https://github.com/Expensify/App/blob/0e0f7b6bdb96f8242d9aa5dc690e4da8b62bda00/src/pages/iou/IOUDetailsModal.js#L123-L133) we have fetched those values from existing `iouReport`.